drivers/CoreEventQueue/include/CoreEventQueue.h

```diff
@@ -18,19 +18,23 @@ class CoreEventQueue : public interface::EventQueue
 
 	void dispatch_forever() final;
 
-	void call(auto f, auto... params) { _event_queue.call(f, params...); }
+	void call(auto f, auto... params) { _event_id = _event_queue.call(f, params...); }
 
 	void call_every(std::chrono::duration<int, std::milli> duration, auto f, auto... params)
 	{
-		_event_queue.call_every(duration, f, params...);
+		_event_id = _event_queue.call_every(duration, f, params...);
 	}
 
+	void cancelLastCall() { _event_queue.cancel(_event_id); }
```

Comment:
I would just use `cancel`

```suggestion
	void cancel() { _event_queue.cancel(_event_id); }
```

in the future we could have `call` return the `id` to be able to `cancel(id)` and share an event queue between different callbacks

what happens if we call `call_every` multiple times? will the queue call all the callback at the desired interval? 

if so, then we should first `cancel` before using `_event_queue.call` or `_event_queue.call_every`

`_event_id` can be an optional to handle the case where cancel is called before `call_every` has been called.